healthtools/views/search_api.py

```diff
@@ -1,12 +1,30 @@
 from flask import Blueprint, request, jsonify
 
 from healthtools.search import run_query
+from healthtools.search.wit_ai import wit_run_query
 
 
 blueprint = Blueprint('search_api', __name__)
 
 
 @blueprint.route('/search', methods=['GET'], strict_slashes=False)
+def search_index(doc_type=None):
+    query = request.args.get('q')
+    result, doc_type = wit_run_query(query, doc_type)
+
+    # Error with wit_run_query (run_query returns false)
+    if(not result):
+        return jsonify({
+            'result': {'hits': [], 'total': 0},
+            'doc_type': doc_type,
+            'status': 'FAILED',
+            'msg': ''  # TODO: Pass run_query message here.
```

Comment:
Why do we have this TODO here?

Reply:
That's me. We should pass an actual message if this fails. Currently we don't have any message being returned so we only know it's failed but not why.

Reply:
Wait, @Awinja-Andela why are these changes reflecting here? I thought I had already written this code.

Reply:
@DavidLemayian i am reusing this code for wit.ai. I am using it just so that we can have two options for searching for the document name.
`using wit.ai `or ` using document.py`
It is either i do this, or i remove  `document.py` completely.